docs/writing_a_parser_plugin/parser_plugin_overview.md

```diff
@@ -0,0 +1,75 @@
+
+
+## Getting started
+
+In principle, it is possible to [develop a "local parser"](references/quick_parser_setup.md) that uses the *nomad-lab* package to parse raw data,
+without changing the NOMAD software itself. This allows a quick start for focusing on the parsing
+of the data itself, but is not relevant for full integration of your new parser into NOMAD.
+Here we are focused on developing parsers that will be integrated into the NOMAD software.
+For this, you will have to [install a development version of NOMAD](references/quick_installing_nomad_dev.md).
+
+
+### Parser organization
+
+The NOMAD parsers can be found within your local NOMAD git repo under
+`dependencies/parsers/`. The parsers are organized into the following individual projects
+(`dependencies/parsers/<parserproject>`) with their own corresponding repositories:
+
+* [atomistic](https://github.com/nomad-coe/atomistic-parsers) - Parsers for output from classical molecular simulations, e.g., from Gromacs, Lammps, etc.
+* [database](https://github.com/nomad-coe/database-parsers) - Parsers for various databases, e.g., OpenKim.
+* [eelsdb](https://github.com/nomad-coe/nomad-parser-eelsdb) - Parser for the EELS database (https://eelsdb.eu/; to be integrated in the database project).
+* [electronic](https://github.com/nomad-coe/electronic-parsers) - Parsers for output from electronic structure calculations, e.g., from Vasp, Fhiaims, etc.
+* [nexus](https://github.com/nomad-coe/nomad-parser-nexus) - Parsers for combining various instrument output formats and electronic lab notebooks.
+* [workflow](https://github.com/nomad-coe/workflow-parsers) - Parsers for output from codes that specialize in workflows, e.g., Aflow, FHI-vibes, QuantumEspressoPhonon, etc.
+
+Within each project folder you will find a `test/` directory, containing the [parser tests](parser_tests.md), and also a directory containing the parsers' source code,
+`<parserproject>parser` or `<parserproject>parsers`, depending on if one or more
+parsers are contained within the project, respectively. In the case of multiple parsers, the files
+for individual parsers are contained within a corresponding subdirectory: `<parserproject>parsers/<parsername>`
+For example, the Quantum Espresso parser files are found in `dependencies/parsers/electronic/electronicparsers/quantumespresso/`.
+
+
+### Setting up your development branches
+
+We will first focus on the case of adding a new parser to an existing parser project.
+[Creating a new parser project](references/advanced_new_parser_project.md) will require a few extra steps.
+The existing parser projects are stored within their own git repositories and then linked
+to the NOMAD software. All current parser projects are available at [nomad-coe](https://github.com/nomad-coe)
+(see also individual links above).
+
+You will first need to create new branches within *both* the NOMAD project and *also* within the corresponding
```

Comment:
Also address getting an account

Reply:
I guess ultimately this will just link to a different section, I will make a TODO.